src/Symfony/Component/Console/Application.php

```diff
@@ -468,15 +468,17 @@ public function setVersion(string $version)
      */
     public function getLongVersion()
     {
+        $name = 'Console Tool';
```

Comment:
I would name the variable `$longVersion` based on the method name. This variable is not only about the name